WeatherApp/src/Components/FetchWeather.vue

```diff
@@ -0,0 +1,257 @@
+<script>
+import axios from "axios";
+import moment from "moment";
+import FetchForecast from "./FetchForecast.vue";
+import FetchHistory from "./FetchHistory.vue";
+import "../css/main.css";
+import "../css/body.css";
+import "../css/photo.css";
+import "../css/panel.css";
+import "../css/responsive.css";
+
+export default {
+  name: "FetchWeather",
+  components: {
+    FetchForecast,
+    FetchHistory,
+  },
+  data() {
+    return {
+      apiKey: "b3241bb7b3f846a4b05132912230903",
```

Comment:
in general, informatiile sensibile precum api keys sunt tinute intr-un fisier numit .env, cu care nu se da commit pe github. 
Te-as sfatui sa extragi toate cheile si sa le pui intr-un astfel de fisier, dupa care sa citesti valorile in vue: https://cli.vuejs.org/guide/mode-and-env.html#environment-variables